e2e/terraform/packer/linux/setup.sh

```diff
@@ -25,15 +25,16 @@ NOMADVERSION=0.9.1
 NOMADDOWNLOAD=https://releases.hashicorp.com/nomad/${NOMADVERSION}/nomad_${NOMADVERSION}_linux_amd64.zip
 NOMADCONFIGDIR=/etc/nomad.d
 NOMADDIR=/opt/nomad
+NOMADPLUGINDIR=/opt/nomad/plugins
 
 # Dependencies
 sudo apt-get install -y software-properties-common
 sudo apt-get update
-sudo apt-get install -y unzip tree redis-tools jq curl tmux awscli nfs-common
+sudo apt-get install -y dnsmasq unzip tree redis-tools jq curl tmux awscli nfs-common
 
 # Numpy (for Spark)
 sudo apt-get install -y python-setuptools
-sudo easy_install pip
+sudo python /usr/lib/python2.7/dist-packages/easy_install.py pip
```

Comment:
I think Ubuntu has a `python-pip` package, which is probably the best way to install it for the system Python.